manifests/yum.pp

```diff
@@ -43,7 +43,8 @@
 #     ``$servers`` entries appropriately
 #
 class simp::yum (
-  Simplib::Netlist                                     $servers,
+  Integer[1, 1]                                        $api_version         = 1,
```

Comment:
If we're going to do this, we might as well make a SemanticVersion Data Type in Simplib and use it.